test/playlist-maker.test.js

```diff
@@ -37,7 +36,7 @@ const videoList = [{
   poster: 'http://media.w3.org/2010/05/video/poster.png'
 }];
 
-QUnit.module('playlist', {
+QUnit.module('playlist-maker', {
```

Comment:
Name change.

Also, a few tests in this file didn't match the style of the majority of tests; so, I made them all uniform.